replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+```
+
+---
+
+### Study 5
+
+> In Study 5, there were 234 participants (mean age = 31.8 years; 107 females, 127 males)
+
+```{r echo = FALSE}
+# Read data in
+suppressMessages(suppressWarnings(df5_demo <- read.csv("csv_files/5_demo.csv", header=T)))
+suppressMessages(suppressWarnings(df5_1 <- read.csv("csv_files/5_Sheet1.csv", header=T)))
+suppressMessages(suppressWarnings(df5_spss <- read.spss("framing/study_5.sav", to.data.frame = TRUE)))
+```
+
+```{r echo = FALSE}
+# Number of subjects
+df5_demo %>%
+  select(subject) %>%
+  summarise(n())
+
+# Number of females vs. males
+df5_demo %>%
+  group_by(gender) %>%
+  summarise(n())
+```
+
+> No participants were excluded from the analyses. We first analyzed responses using a median split on participants’ dieting behavior. Nondieters rated the fries as more fattening when they thought about a small account (M = 8.64, 95% CI = [8.20, 9.07]) than when they thought about a large account (M = 7.80, 95% CI = [7.21, 8.38]),
+
+
+```{r echo = FALSE}
+# Nondieters & Small account : Fattening
+df5_spss %>%
+  filter(split == 0, week == 0) %>%
+  summarise(mean.fattening.nondieters.smallaccount = mean(fattening), CI.low = mean(fattening) - (1.96 * Sem(fattening)), CI.high = mean(fattening) + (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+
+# Nondieters & Large account : Fattening
+df5_spss %>%
+  filter(split == 0, week == 1) %>%
+  summarise(mean.fattening.nondieters.largeaccount = mean(fattening), CI.low = mean(fattening) - (1.96 * Sem(fattening)), CI.high = mean(fattening) + (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+```
+
+> but this was not true for dieters (small account: M = 8.86, 95% CI = [8.29, 9.43]; large account: M = 9.13, 95% CI = [8.66, 9.60]).
+
+```{r echo = FALSE}
+
+# Dieter & Small account : Fattening
+df5_spss %>%
+  filter(split == 1, week == 0) %>%
+  summarise(mean.fattening.dieters.smallaccount = mean(fattening), CI.low = mean(fattening) - (1.96 * Sem(fattening)), CI.high = mean(fattening) + (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+
+
+# Dieter & Large account : Fattening
+df5_spss %>%
+  filter(split == 1, week == 1) %>%
+  summarise(mean.fattening.dieters.largeaccount = mean(fattening), CI.low = mean(fattening) - (1.96 * Sem(fattening)), CI.high = mean(fattening) + (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+
+```
+
+
+
+```{r echo = FALSE}
+df5_nondiet_small <- df5_spss %>%
+  filter(split == 0, week == 0) %>%
+  summarise(mean.fattening.nondieters.smallaccount = mean(fattening), CI = (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+
+df5_nondiet_large <- df5_spss %>%
+  filter(split == 0, week == 1) %>%
+  summarise(mean.fattening.nondieters.largeaccount = mean(fattening), CI = (1.96 * Sem(fattening))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (5) - non-dieters
+```
+
```

Comment:
I see that for the last few studies you do assign a new name for filtered out datasets - I guess its optional if you don't do it, but perhaps keep whichever decision you make consistent throughout the script.